roles/app/tasks/main.yml

```diff
@@ -130,12 +130,22 @@
   tags:
      - deploy
 
-- name: create slumber api consumer django user with proper permissions
+- name: check if slumber api consumer django user exists
+  become_user: "{{ rtd_user }}"
+  command: "{{ rtd_virtualenv }}/bin/python {{ rtd_root }}/manage.py shell --settings=readthedocs.settings.managed | grep True"
+  args:
+    stdin: "from django.contrib.auth.models import User; User.objects.filter(username='{{ SLUMBER_USERNAME }}').exists()"
+  register: build_user_available
```

Comment:
we need to define this as a default, otherwise if this fails, the check for the next command won't find the variable

Reply:
Even if the commands fail since it has ignore_errors set to true it should be still accounted.

Reply:
Ok, it works